### ISSUE TITLE
[Feature] [Visual comparisons] Snapshot path and name

### ISSUE THREAD

**Issue by LiutskoOlga** [microsoft/playwright#13296]
Hello amazing team. My team really need some small improvements from you:
1. Could I ask you about add possibility to put snapshot's picture in some common folder instead of spec name folder. Because better to save some snapshots in one place and not duplicate it in such different folders. _(for example: I have 3 spec file spec1.ts, spec2.ts, spec3.ts -it's different tests/suites but as expected I have one file comprising, but I need to keep three folders with one file -> spec1/sample.png; spec2/sample.png; spec3/sample.png)_
2. And could we please skip the automatic generation name of snapshots for different OS _(for example: I create snapshot step.png, but MacOS it saved as step-drawing.png, linux - step-linux.png_). So it's a little bit complexing for us to run tests on different system and keep updating all this bunch of pictures

Thank you so much!

**Comment by vitalics** [comment]
@LiutskoOlga I thing that your feature request is not relevant. Let me explain my minds

you can override `testinfo.snapshotSuffix` and `testInfo.snapshotDir` properties to provide custom snapshot path

example
``` ts
import {resolve} from 'path';

test('demo', async ({ page }, testInfo) => {

  await page.goto('https://www.google.com');
  testInfo.snapshotSuffix = ''; // removes process.platform. See https://playwright.dev/docs/api/class-testinfo#test-info-snapshot-suffix
  testInfo.snapshotDir = resolve(__dirname, 'a.ts-snapshots'); //  in current test folder creates new folder named "a.ts-snapshots"

  expect(await page.screenshot()).toMatchSnapshot('google.png') // generates '<path-to-test>/a.ts-snapshots/google.png' 
});
```

@pavelfeldman @mxschmitt Correct me if Im wrong, but it's not recommended since it can affect other tests. 

As for me, the best solution here - isolate your test as much as possible.

**Comment by Marcin3** [comment]
Can we split this feature? For me first point is more important and maybe in this way could be delivered faster.

**Comment by aslushnikov** [comment]
I'll merge this in https://github.com/microsoft/playwright/issues/7792 that should be fixed by the upcoming 1.28

**Comment by Marcin3** [comment]
Thanku you.
